embulk-standards/src/main/java/org/embulk/standards/JsonParserPlugin.java

```diff
@@ -83,6 +85,12 @@ public String getString() {
         Optional<SchemaConfig> getSchemaConfig();
     }
 
+    public interface OptionalColumnConfig extends Task {
+        @Config("json_pointer")
```

Comment:
Not very confident, but we may want to call it a kind of `relative`, IIUC?

Reply:
Yes, it can be relative if `__experimental__json_pointer_to_root` is specified.. so something like `relative_json_pointer_from_root` might be better, which is long, though.

Reply:
Renamed to `relative_json_pointer_from_root`. How about this?
ref: 7672bae

Reply:
Sounds reasonable. At the same time, it could be worth looking into https://tools.ietf.org/id/draft-handrews-relative-json-pointer-00.html while it's still draft.